src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/payment/processor/client/payhub/PayHubClientTest.java

```diff
@@ -101,60 +98,7 @@ public void should_return_PayHubClientException_for_badRequest() throws IOExcept
         Throwable throwable = catchThrowable(() -> client.createPayment(s2sToken, getPaymentRequest()));
 
         // then
-        assertThat(throwable).isInstanceOf(PayHubClientException.class);
-
-        // and
-        PayHubClientException exception = (PayHubClientException) throwable;
-
-        assertThat(exception.getStatus()).isEqualTo(BAD_REQUEST);
-    }
-
-
-    @Test
-    public void should_return_PayHubClientException_for_conflict() throws IOException {
```

Comment:
This case is still valid, isn't it? 409 scenario can still happen in the client.

Reply:
there's no `PayHubClientException` now and feign client throws a status code specific exception. Don't want to got through all possible status code / exceptions in tests.
Just happy path and error scenario.

Reply:
this is payhub feign client. it does not rely on specific codes to be returned. as @rkondratowicz says - happy and error scenarios are enough

Reply:
Then it would be best to change the name of the test method to contain "error response", instead of specific error.